blackbox/docs/general/builtins/window-functions.rst

```diff
@@ -25,43 +25,98 @@ The ``OVER`` clause defines the ``window`` containing the appropriate rows
 which will take part in the ``window function`` computation.
 
 An empty ``OVER`` clause defines a ``window`` containing all the rows in the
-result set.
+result set. 
 
 Example::
 
-   cr> select name, avg(price) OVER() from articles order by name;
-   +------------------------------+--------------------+
-   | name                         | avg(price) OVER () |
-   +------------------------------+--------------------+
-   | Infinite Improbability Drive | 18375.317556142807 |
-   | Kill-o-Zap blaster pistol    | 18375.317556142807 |
-   | Starship Titanic             | 18375.317556142807 |
-   | Towel                        | 18375.317556142807 |
-   +------------------------------+--------------------+
-   SELECT 4 rows in set (... sec)
+   cr> select dept_id, row_number() OVER() from employees order by 2, 1;
```

Comment:
I think this ordering won't be deterministic. `row_number() -> 1` could go to any row since we don't have a guranteed natural ordering.